go/fn/go.mod

```diff
@@ -12,11 +12,12 @@ require (
 	sigs.k8s.io/kustomize/kyaml v0.13.7-0.20220418212550-9d5491c2e20c
 )
 
+require github.com/go-errors/errors v1.4.2
```

Comment:
Please group this with deps above.

Reply:
Good catch! didn't pay attention to this.